RELEASE/scripts/autoscend/auto_quest_level_12.ash

```diff
@@ -48,70 +48,19 @@ int auto_warSideQuestsDone()
 	return sidequests_done;
 }
 
-int auto_warSideQuestsState()
+WarPlan auto_warSideQuestsState()
 {
 	// returns a bitwise indicating current completion state of the war sidequests.
```

Comment:
according to malibu this should say fields. bitwise is the name of the operators that handle fields

Reply:
malibu is right, that is true, also this comment is out of date anyway because it no longer returns a bitfield at all. will change.